tests/pulse/test_parametrized_evolution.py

```diff
@@ -36,8 +36,48 @@ class MyOp(qml.RX):  # pylint: disable=too-few-public-methods
     has_diagonalizing_gates = False
 
 
+def amp(p, t):
+    return p * t
+
+
+H = qml.PauliX(1) + amp * qml.PauliZ(0) + amp * qml.PauliY(1)
+params = [0.5, 0.5]
+
+example_pytree_evolutions = [
+    qml.pulse.ParametrizedEvolution(H, params, t=0.5),
```

Comment:
Do you want to also test one that is initialized but without `params` and `t`? Since it came up in working with the `equal` function.

Reply:
Yep! ✅